renormalise_d8LF.py

```diff
@@ -0,0 +1,8 @@
+def vol_normalise(cat, scale):
```

Comment:
This is a bit generic to my taste.  Can we replace scale with fdelta, add a docstring pointing to the equation in Tamsyn's paper and check that the fdelta usage is consistent with that used there.

Reply:
vol_normalise -> lumfn_d8_renormalise